src/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import * as cx from "classnames";
+import FlexView from "react-flexview";
+import { props, t } from "../utils";
+
+export type RadioOption = {
+  label: string;
+  value: string;
+};
+
+export type RadioGroupRequiredProps = {
+  /** name for the group */
+  name: string;
+  /** value */
+  value?: string;
+  /** onChange */
+  onChange: (value: string) => void;
```

Comment:
is it the same as `Dropdown`? are we using the string value instead of the option?

Reply:
`Dropdown` is more complex: `export type Value = SelectNS.Option | SelectNS.Options | SimpleValue;`

I kept things as simple as possible here (only "simple value", and harcoded to `string`). Do you think we should add more flexibility?

Reply:
no, it's enough for now (also this is controversial in `Dropdown` as well...)